README.md

```diff
@@ -0,0 +1,34 @@
+## nipype-workflows
+
+This repository consists of neuroimaging workflows writtten in nipype with a focus on the Portable Batch System (PBS) job scheduler.
+It provides tools and documentation for BMC users using the services of UF's Researchcomputing (RC) group.
```

Comment:
How about this:

It provides tools and documentation for neuroscientists to using the services of UF's Research Computing (RC) group.